attest/untrusted/src/lib.rs

```diff
@@ -7,14 +7,22 @@ use mc_attest_core::{
     EpidGroupId, PibError, PlatformInfoBlob, ProviderId, Quote, QuoteError, QuoteNonce,
     QuoteSignType, Report, SgxError, SigRL, TargetInfo, UpdateInfo,
 };
-#[cfg(not(feature = "sgx-sim"))]
-use mc_sgx_dcap_ql::QeTargetInfo;
 use mc_sgx_dcap_types::QlError;
 use mc_sgx_types::{
     sgx_calc_quote_size, sgx_get_extended_epid_group_id, sgx_get_quote,
     sgx_report_attestation_status, sgx_status_t,
 };
 
+cfg_if::cfg_if! {
+    if #[cfg(feature = "sgx-sim")] {
+        mod sim;
+        pub type DcapQuotingEnclave = crate::sim::SimQuotingEnclave;
+    } else {
+        mod hw;
+        pub type DcapQuotingEnclave = crate::hw::HwQuotingEnclave;
+    }
+}
+
```

Comment:
This pattern mimics that of `mc-attest-net` and the RaClient. 
Future plan is to replace the RaClient with the `DcapQuotingEnclave` where appropriate.